src/widgets/helper/ChannelView.cpp

```diff
@@ -1983,6 +1984,14 @@ void ChannelView::addContextMenuItems(
         }
     }
 
+    if (event->modifiers() == Qt::ShiftModifier &&
+        !layout->getMessage()->id.isEmpty())
+    {
+        menu->addAction("Copy message id", [layout] {
```

Comment:
```suggestion
        menu->addAction("Copy message ID", [layout] {
```

Reply:
I actually thought this looked weird but since `message` wasn't capitalized in all options I ignored it, but this ID capitalization is correct.

Reply:
Pretty much what Felan said. I was following how it was elsewhere to avoid different stylings. If it were up to me I would have capitalized to `ID`. As for the menu action names, I would also capitalize the names to be `Copy Message` and so on, but again, I was following existing styling.

Reply:
Identification is never abbreviated to `id`, it should be `ID`. Id is a word, ID is the abbreviation.